flake.nix

```diff
@@ -7,14 +7,30 @@
 
   outputs = { self, nixpkgs, flake-utils, ocaml-overlays }:
     flake-utils.lib.eachDefaultSystem (system:
-      let pkgs = import nixpkgs { inherit system; overlays = [ocaml-overlays.overlay]; };
-      in
-      {
+      let
+        pkgs = import nixpkgs {
+          inherit system;
+          overlays = [ ocaml-overlays.overlay ];
+        };
+      in {
         devShell = (pkgs.mkShell {
           shellHook = ''
             npm install
             export PATH="node_modules/.bin:_build/default/src/bin:$PATH"
           '';
+          buildInputs = with pkgs; [
+            # Make develop life easier
+            # General tooling
+            docker
```

Comment:
I'm not sure about docker, it's good to have it scoped to the project but I'm not sure how this will interact with globally installed docker which is probably the norm.

Reply:
>how this will interact with globally installed docker

The one provided in this flake will not be used, and `which docker` will result to use the globally installed one.

>[...] which is probably the norm

I don't think relying on "_what is probably the norm_" is a good thing for the future.
From my personal experience, I already gave up to participate to some open-source projects because of a prerequisite list as long as my arm is... Hence, if we can provide a "_nix, it's all it takes to participate_" I think this is a good idea 😄 

This could also avoid us trouble like:
> Oh, this won't work with `whateverbin 2.X`, so please find a way to install the `1.9.x` version

since we directly provide the working ones ^^

Anyway, this is just a suggestion, since our prerequisite isn't long, I don't see a real trouble here.

Reply:
> The one provided in this flake will not be used, and which docker will result to use the globally installed one.

If this is true I don't think it's a problem to do this :)